### PR TITLE
python3Packages.pymfy: init at 0.9.4

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python client for the Somfy Open API

https://github.com/tetienne/somfy-open-api

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113799` at 5a8b3a93 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python39Packages.pymfy</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/2c3bb6d62418345cd676b23e0635c467">python37Packages.pymfy</a></li></li>
    <li><a href="https://gist.github.com/b17edf6429f7bb57d9fe5d3fed4eab9c">python38Packages.pymfy</a></li></li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
@fabaff please resolve the merge conflict.

**Comment by fabaff** [comment]
> @fabaff please resolve the merge conflict.

@SuperSandro2000, done.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5708,7 +5708,11 @@ in {
 
   pymetno = callPackage ../development/python-modules/pymetno { };
 
+<<<<<<< HEAD
   pymitv = callPackage ../development/python-modules/pymitv { };
+=======
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:5713]: You did not cleanly resolve the merge conflict.

Comment (fabaff) [reply pkgs/top-level/python-packages.nix]: Looks like that the previous forced pull didn't pick-up the changes.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 113799` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.pymfy</li>
    <li>python38Packages.pymfy</li>
    <li>python39Packages.pymfy</li>
  </ul>
</details>

**Pull request merged**